### PR TITLE
Add voting stars feature w/ example

### PR DESCRIPTION
**Pull request by tholex**

### CODE REVIEW

File: src/NotificationView/NotificationItem.jsx [hunk 1]
```diff
@@ -18,6 +18,20 @@ class NotificationItem extends React.Component {
     this.props.handleDismiss && this.props.handleDismiss(e, this.props.id);
   }
 
+  renderVotingBar() {
+    return (<div className="tui-notification-voting-bar">
+      {[5,4,3,2,1].map((num) => {
```
Comment (learykara) [inline src/NotificationView/NotificationItem.jsx]: why reverse this array with `direction: rtl`?

File: src/NotificationView/notifications.less [hunk 2]
```diff
@@ -173,3 +173,25 @@
...
+  &:hover g {
+    fill: @blue;
+  }
+
+  + .tui-voting-star {
+    padding-right: 5px;
+  }
+}
+
+.tui-voting-star:hover ~ .tui-voting-star g {
```
Comment (learykara) [inline src/NotificationView/notifications.less]: ah, is above question for this?

File: src/NotificationView/NotificationItem.jsx (hunk 1 above)
Comment (tholex) [inline src/NotificationView/NotificationItem.jsx]: This is so you can use `star:hover + star` to highlight all the "1,2,3" stars when you hover on the 4th

File: src/NotificationView/NotificationItem.jsx [hunk 3]
```diff
@@ -30,6 +44,7 @@ class NotificationItem extends React.Component {
         <time className="tui-notification-time">{timeDifference}</time>
         {debugThis && <pre>{JSON.stringify(this.props)}</pre>}
         <p className="tui-notification-message">{this.props.message}</p>
+        {this.props.votable && this.props.votable_url && this.renderVotingBar()}
```
Comment (learykara) [inline src/NotificationView/NotificationItem.jsx]: `this.props.votable` is redundant -- can just check for `!!this.props.votable_url`

File: src/NotificationView/notifications.less (hunk 2 above)
Comment (tholex) [inline src/NotificationView/notifications.less]: ya

**Comment by learykara** [comment]
:+1:

**Pull request merged**